### ISSUE TITLE
ogg streams crash

### ISSUE THREAD

**Issue by hurzl** [segler-alex/RadioDroid#40]
Especially Deutschlandradio

**Comment by hurzl** [comment]
```

05-16 11:50:39.913 25067-28679/net.programmierecke.radiodroid2 E/MediaPlayer: attachNewPlayer called in state 4
05-16 11:50:39.913 25067-28679/net.programmierecke.radiodroid2 E/PLAY: java.lang.IllegalStateException
05-16 11:50:39.973 25067-28679/net.programmierecke.radiodroid2 E/AndroidRuntime: FATAL EXCEPTION: Thread-16211
                                                                                 Process: net.programmierecke.radiodroid2, PID: 25067
                                                                                 java.lang.RuntimeException: Can't create handler inside thread that has not called Looper.prepare()
                                                                                     at android.os.Handler.<init>(Handler.java:200)
                                                                                     at android.os.Handler.<init>(Handler.java:114)
                                                                                     at android.widget.Toast$TN.<init>(Toast.java:346)
                                                                                     at android.widget.Toast.<init>(Toast.java:102)
                                                                                     at android.widget.Toast.makeText(Toast.java:260)
                                                                                     at net.programmierecke.radiodroid2.PlayerService$3.run(PlayerService.java:365)
                                                                                     at java.lang.Thread.run(Thread.java:818)
05-16 11:50:40.754 25067-28276/net.programmierecke.radiodroid2 I/PROXY: stop()

```

**Comment by Wikinaut** [comment]
Which RadioDroid version ?

**Comment by hurzl** [comment]
0.14.6

**Comment by Wikinaut** [comment]
@hurzl 
Deutschlandradio Kultur has three OGG streams, have you tested all three ?

* 48-128 kBit/s VBR -> works for me on HUAWEI Y201, and HUAWEI P8 LITE
* 56 kBit/s ->  works for me on HUAWEI Y201, and HUAWEI P8 LITE
* 33 kBit/s ->  works for me on HUAWEI P8 LITE. On HUAWEI Y201 I see "Bereite Stream vor..." and it does not play.

I noticed problems with OGG now and then with several devices and software, maybe a general problem for older Android versions. Sometimes, closing and restart of RadioDroid, or the smartphone helped.

Loading and buffering of the slow streams can take 2..3 minutes until audio is played! But works.

Here are the Stream Urls for all quality levels:
http://www.deutschlandradio.de/deutschlandradio-im-internet-mit-ogg-vorbis-stream.508.de.html

**Comment by hurzl** [comment]
Ich probiere "OGG HQ". Mehr ist an Text nicht zu erkennen. Was hat die IllegalStateException oben damit zu tun?

**Comment by Wikinaut** [comment]
@hurzl 
I justed wanted to point out (and confirm in that way), that playing OGG sometimes works and sometimes does not work. I cannot debug the code at the moment, but on my HUAWEI Y201 and the 33 kBits/s stream (which is not playing there), when I try to stop the play action, RadioDroid crashes on that device.

**Comment by segler-alex** [comment]
it is always possible to "share" stations with other mediaplayers. for example VLC. it will play any station. for the time beeing i am using the android internal mediaplayer and it does not play all streams very good :(
maybe i will add FFMPEG at some time. but this will need much work

**Comment by segler-alex** [comment]
fixed crash in 0.19
